pkg/packer/manifests/vsphere/packer.json.tmpl

```diff
@@ -2,7 +2,7 @@
     "variables": {
         "ansible_extra_vars": "",
         "build_timestamp": "{{timestamp}}",
-        "vm_name": "konvoy-ova-{{user `build_name`}}{{user `build_name_extra`}}-{{user `kubernetes_full_version` }}-{{user `build_timestamp`}}",
```

Comment:
build_name_extra is already added by the go code. so removing repeatation here.  https://github.com/mesosphere/konvoy-image-builder/blob/main/pkg/app/config.go#L195

Reply:
What will the resulting name be? Doesn't the `-release` come from `build_name_extra` which are we now removing?

Reply:
I am planning to change the TC job to add the branch_name in the `build_name_extra` 
```
cat <<EOF > release-overrides.yaml
build_name_extra: release-%teamcity.build.branch%
EOF
```
the go code adds the `build_name_extra` so we dont need to add it explicitly in packer template.
https://github.com/mesosphere/konvoy-image-builder/blob/main/pkg/app/config.go#L98

The resulting template name will look like: `konvoy-ova-vsphere-rhel-79-release-v1.13.2-1.22.8+fips.0-1652304962`

Reply:
Thanks 👍,
[I see its not in AMI either](https://github.com/mesosphere/konvoy-image-builder/blob/90c84fc9d9403e150bf1075c277f17c1f5e33ce7/pkg/packer/manifests/aws/packer.json.tmpl#L60)